Source/WebKit/WebProcess/InjectedBundle/glib/InjectedBundleGlib.cpp

```diff
@@ -49,6 +53,11 @@ bool InjectedBundle::initialize(const WebProcessCreationParameters&, RefPtr<API:
     }
 
     initializeFunction(toAPI(this), toAPI(initializationUserData.get()));
+
+#if PLATFORM(WPE)
+    WTF::registerInspectorResources();
+#endif
```

Comment:
I don't understand why doing it here, it means that any browser using injected bundle will load the resources unconditionally. Why don't we call this from Internals::openDummyInspectorFrontend?

Reply:
Oops. I wanted to do it in the injected bundle of WebKitTestRunner. I'm not confident the web process of inspector is always same with the inspected window. It works in openDummyInspectorFrontend. Will do that.